### PR TITLE
Feature/IESP-33: implementation of service to update polling frequency and time interval

### PR DESCRIPTION
**Pull request by angelaantoniaceaa**
…lling time interval

### CODE REVIEW

File: src/main/java/it/pagopa/interop/probing/probingapi/mapstruct/dto/UpdateEserviceFrequencyDto.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import java.time.OffsetTime;
+import java.util.UUID;
+
+import javax.validation.constraints.NotNull;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import lombok.Data;
+
+@Data
```
Comment (nttdata-rtorsoli) [inline src/main/java/it/pagopa/interop/probing/probingapi/mapstruct/dto/UpdateEserviceFrequencyDto.java:12]: Add @Builder

**Review by nttdata-rtorsoli — APPROVED** [review]

File: src/main/java/it/pagopa/interop/probing/probingapi/service/EserviceService.java [hunk 2]
```diff
@@ -1,22 +1,42 @@
...
 
-    /**
-     * Updates the state of the e-service identified by the input eserviceId and versioneId
-     * @param  inputData the input data DTO containing the e-service id, version id and the probing new state
-     * @throws EserviceNotFoundException if the e-service isn't found in the database
-     * */
-    void updateEserviceState(UpdateEserviceStateDto inputData) throws EserviceNotFoundException;
+	/**
+	 * Updates the state of the e-service identified by the input eserviceId and
+	 * versioneId
```
Comment (galales) [inline src/main/java/it/pagopa/interop/probing/probingapi/service/EserviceService.java:12]:
```suggestion
	 * versionId
```

File: src/main/java/it/pagopa/interop/probing/probingapi/service/EserviceService.java [hunk 3]
```diff
@@ -1,22 +1,42 @@
...
+	 * @param inputData the input data DTO containing the e-service id, version id
+	 *                  and the probing new state
+	 * @throws EserviceNotFoundException if the e-service isn't found in the
+	 *                                   database
+	 */
+	void updateEserviceState(UpdateEserviceStateDto inputData) throws EserviceNotFoundException;
+
+	/**
+	 * Updates the probing state of the e-service identified by the input eserviceId
+	 * and versioneId
```
Comment (galales) [inline src/main/java/it/pagopa/interop/probing/probingapi/service/EserviceService.java:23]:
```suggestion
	 * and versionId
```

File: src/main/java/it/pagopa/interop/probing/probingapi/service/EserviceService.java [hunk 4]
```diff
@@ -1,22 +1,42 @@
...
 
-    /**
-     * Updates the probing state of the e-service identified by the input eserviceId and versioneId
-     * @param  inputData the input data DTO containing the e-service id, version id and the probing enabling/disabling
-     * @throws EserviceNotFoundException if the e-service isn't found in the database
-     * */
-    void updateEserviceProbingState(UpdateEserviceProbingStateDto inputData) throws EserviceNotFoundException;
+	/**
+	 * Updates the frequency, pollingStartTime and pollingStartTime of the e-service
+	 * identified by the input eserviceId and versioneId
```
Comment (galales) [inline src/main/java/it/pagopa/interop/probing/probingapi/service/EserviceService.java:34]:
```suggestion
	 * identified by the input eserviceId and versionId
```

**Review by galales — APPROVED** [review]

**Pull request merged**